.travis.yml

```diff
@@ -35,7 +35,9 @@ matrix:
   - rust: nightly
     script:
       # Run tests/builds
-      - sudo apt install build-essential
+      # A regression happened in rust-nightly 1.46.0. Therefore, we set the previous toolchain
+      # to compile & run the tests with rust 1.45.0.
+      - rustup override set nightly-2020-06-04
```

Comment:
As I mentioned by our previous call, I would use [the toolchain file](https://github.com/rust-lang/rustup#the-toolchain-file) instead.

The reasons ara:

1. Using a specific toolchain in travis configuration means we're hide this information: it's not clear from the repo at first sight we have a different toolchain here. Instead we can spot immediately a toolchain file in the root.

2. Since the toolchain is set only for travis, it means if check out the repo and build it locally I will use my local default toolchain, and therefore I might have a different result than travis: with a toolchain file the result will be the same.

Reply:
Makes sense, Will re-do it